examples/employees/README.md

```diff
@@ -0,0 +1,15 @@
+Example queries using [employees database](https://github.com/vrajmohan/pgsql-sample-data.git).
```

Comment:
This is awesome! We could even turn this into a test later, ref https://github.com/max-sixty/prql/issues/219